src/OpenLoco/Windows/VehicleList.cpp

```diff
@@ -27,36 +51,850 @@ namespace OpenLoco::Ui::Windows::VehicleList
+        [[maybe_unused]] auto args = FormatArguments::common(company->name);
+
+        static constexpr string_id typeToCaption[] = {
+            StringIds::stringid_trains,
+            StringIds::stringid_buses,
+            StringIds::stringid_trucks,
+            StringIds::stringid_trams,
+            StringIds::stringid_aircraft,
+            StringIds::stringid_ships,
+        };
+
+        // Basic frame widget dimensions
+        self->widgets[Widx::frame].right = self->width - 1;
+        self->widgets[Widx::frame].bottom = self->height - 1;
+
+        self->widgets[Widx::panel].right = self->width - 1;
+        self->widgets[Widx::panel].bottom = self->height - 1;
+
+        self->widgets[Widx::caption].right = self->width - 2;
+        self->widgets[Widx::caption].text = typeToCaption[self->current_tab];
+
+        self->widgets[Widx::close_button].left = self->width - 15;
+        self->widgets[Widx::close_button].right = self->width - 3;
+
+        self->widgets[Widx::scrollview].right = self->width - 4;
+        self->widgets[Widx::scrollview].bottom = self->height - 14;
+
+        // Reposition table headers
+        self->widgets[Widx::sort_name].right = std::min(self->width - 4, 313);
+
+        self->widgets[Widx::sort_profit].left = std::min(self->width - 4, 314);
+        self->widgets[Widx::sort_profit].right = std::min(self->width - 4, 413);
+
+        self->widgets[Widx::sort_age].left = std::min(self->width - 4, 414);
+        self->widgets[Widx::sort_age].right = std::min(self->width - 4, 478);
+
+        self->widgets[Widx::sort_reliability].left = std::min(self->width - 4, 479);
+        self->widgets[Widx::sort_reliability].right = std::min(self->width - 4, 545);
+
+        // Reposition company selection
+        self->widgets[Widx::company_select].left = self->width - 28;
+        self->widgets[Widx::company_select].right = self->width - 3;
+
+        // Set header button captions.
+        self->widgets[Widx::sort_name].text = self->sort_mode == SortMode::Name ? StringIds::table_header_name_desc : StringIds::table_header_name;
+        self->widgets[Widx::sort_profit].text = self->sort_mode == SortMode::Profit ? StringIds::table_header_monthly_profit_desc : StringIds::table_header_monthly_profit;
+        self->widgets[Widx::sort_age].text = self->sort_mode == SortMode::Age ? StringIds::table_header_age_desc : StringIds::table_header_age;
+        self->widgets[Widx::sort_reliability].text = self->sort_mode == SortMode::Reliability ? StringIds::table_header_reliability_desc : StringIds::table_header_reliability;
+
+        setTransportTypeTabs(self);
+    }
+
+    // 0x004C211C
+    static void draw(window* self, Gfx::drawpixelinfo_t* dpi)
+    {
+        self->draw(dpi);
+        drawTabs(self, dpi);
+
+        // Draw company owner image.
+        auto company = CompanyManager::get(self->number);
+        auto competitor = ObjectManager::get<CompetitorObject>(company->competitor_id);
```

Comment:
`competitorObj` I feel that calling it competitor feels like it would represent another company but it represents this companies object.